src/commands/install.ts

```diff
@@ -0,0 +1,54 @@
+import Vorpal from "vorpal";
+//import { execSync } from "child_process";
+//import { InstallUtils } from "../classes/install-utils";
+
+/**
+ * Installs software with a given flag, or activates a wizard to give options
+ * 
+ * @param args gets an object that can contain the software(string)'s name and options
+ * options include:
+ * software(string)
+ */
+async function action(args){
+  let software : string = args.options.software;
+  if(!software){
+    try {
+      const softwareResult = await this.prompt({
+        type : 'list',
+        name : 'software',
+        choices : ['NodeJs', 'GitHub', 'Git CLI', 'Maven', 'Java Development Kit 8', 'Java Development Kit 11', 'Homebrew'],
```

Comment:
This list will grow, recommend storing it as a variable in a less crowded place of the file (near the top perhaps)

Reply:
this is a placeholder either way, as these strings will be included in the types.ts as global constants in another issue

Reply:
that sounds fine